orangecontrib/text/import_documents.py

```diff
@@ -211,8 +222,95 @@ def make_text_data(self):
                         text_data.category, text_data.content)
 
 
+class ConlluReader(Reader):
+    TextData = namedtuple(
+        "Text",
+        ["name", "path", "ext", "category", "doc_id", "content"]
+    )
+
+    ext = [".conllu"]
+
+    def __init__(self, path):
+        super().__init__(path)
+        self.tokens = None
+        self.pos = None
+        self.ner = None
+
+    @staticmethod
+    def parse_ner(tokens):
+        entities = []
+        temp_ner = []
+        for token in tokens:
+            if token["misc"] is None or "NER" not in token["misc"]:
+                continue
+            # "0" means the token is not named entity
+            if token["misc"]["NER"] != "O":
+                # lemma?
+                temp_ner.append(token["lemma"])
+            elif temp_ner:
+                entities.append(" ".join(temp_ner))
+                temp_ner = []
+        if temp_ner:
+            entities.append(" ".join(temp_ner))
+        return entities
+
+    def read_file(self):
+        content = []
+        file = open(self.path, "r")
```

Comment:
According to the documentation, conllu is utf-8 encoded so replace this line with 
```
file = open(self.path, "r", encoding="utf-8")
```
to fix the Windows issue.